client/luigi-client.d.ts

```diff
@@ -612,10 +612,19 @@ export type getEventData = () => Context;
  */
 export function getContext(): Context;
 export type getContext = () => Context;
-
+/**
+ * Set search query parameters to Luigi core. It will be added to url.
```

Comment:
```suggestion
 * Sets search query parameters in Luigi Core. The search parameters will be added to the URL.
```

Sorry, I missed this one, but I think it should be the same as https://github.com/SAP/luigi/pull/2409/files#diff-ea2419053062f053bee8d5e4a11c1df366970c29d50601d12ac2fe1ceebcb149R405 .